modules/tests/photons_control_tests/test_clean.py

```diff
@@ -0,0 +1,152 @@
+# coding: spec
+
+from photons_control.clean import CleanConfig, CleanCycle
+
+from photons_app import helpers as hp
+
+from photons_messages import DeviceMessages, LightMessages, DiscoveryMessages
+
+from photons_products import Products
+
+import pytest
+
+devices = pytest.helpers.mimic()
+
+light1 = devices.add("light1")(
+    "d073d5000001",
+    Products.LCM3_A19_CLEAN,
+    hp.Firmware(3, 70),
+    value_store=dict(
+        power=0,
+        color=hp.Color(0, 1, 0.3, 2500),
+    ),
+)
+
+light2 = devices.add("light2")(
+    "d073d5000002",
+    Products.LCM3_A19_CLEAN,
+    hp.Firmware(3, 70),
+    value_store=dict(
+        power=65535,
+        indication=True,
```

Comment:
so I don't think the mimic supports hev messages. You'd have to add an operator over in https://github.com/delfick/photons/tree/main/modules/photons_app/mimic/operators

Reply:
though the operator just means you can add attribute tests too. but otherwise these tests are actually really good :)